schema/project-version.js

```diff
@@ -35,6 +35,13 @@ class ProjectVersion extends BaseModel {
       .eager('project');
   }
 
+  static getAll(projectId) {
+    return this.query()
+      .where(builder => {
+        return builder.where({ projectId });
+      });
```

Comment:
There's no need for the builder function wrapper here. It can just be `return this.query().where({ projectId });`